src/main/msp/msp_protocol.h

```diff
@@ -55,15 +55,11 @@
 
 #pragma once
 
-/* Protocol numbers used both by the wire format, config system, and
-   field setters.
-*/
-
+// Protocol numbers used both by the wire format, config system, and field setters.
 #define MSP_PROTOCOL_VERSION                0
 
-#define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   47 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
while it seems obvious to us, why remove these comments. maybe in years to come when we all dead, someone forget workflow.